src/ui/pages/NotificationDetails/components/ReceiveCredential/ReceiveCredential.tsx

```diff
@@ -82,7 +85,15 @@ const ReceiveCredential = ({
           notificationDetails.a.d as string
         );
 
-      setCredDetail(credential);
+      const identifier = identifiersData.find((identifier) => {
+        identifier.id === credential.identifierId;
+      });
+
+      const identifierType = identifier?.multisigManageAid
+        ? IdentifierType.Group
+        : IdentifierType.Individual;
+
+      setCredDetail({ ...credential, identifierType });
```

Comment:
It doesn't make sense to not get it from here, or we would've just kept it in the core. What we should do here, is add `Omit` for the cred detail type to remove the identifier type.